go/cmd/aggregate-crls/crl-auditor.go

```diff
@@ -98,6 +101,23 @@ func (auditor *CrlAuditor) FailedVerifyUrl(issuer storage.Issuer, crlUrl *url.UR
 	})
 }
 
+func (auditor *CrlAuditor) FailedOlderthanPrevious(issuer storage.Issuer, crlUrl *url.URL, dlAuditor *DownloadAuditor, previous time.Time, this time.Time) {
```

Comment:
s/Olderthan/OlderThan 
(same with the test function).

Reply:
Thanks!